src/App.js

```diff
@@ -1,41 +1,13 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import { Routes, Route, Link } from "react-router-dom";
+import {Navbar} from "./components/index"
+import {Searchbar} from "./components/index"
```

Comment:
could import Navbar and searchbar at once

Reply:
ohkay rajat thanks